atoma-daemon/src/handlers/almost_filled_stacks.rs

```diff
@@ -0,0 +1,109 @@
+use atoma_state::types::Stack;
+use axum::{
+    extract::{Path, State},
+    http::StatusCode,
+    routing::get,
+    Json, Router,
+};
+use tracing::error;
+use utoipa::OpenApi;
+
+use crate::DaemonState;
+
+pub const ALMOST_FILLED_STACKS_PATH: &str = "/almost_filled_stacks";
+
+#[derive(OpenApi)]
+#[openapi(
+    paths(almost_filled_stacks_fraction_get, almost_filled_stacks_get),
+    components(schemas(Stack))
+)]
+pub(crate) struct AlmostFilledStacksOpenApi;
+
+pub fn almost_filled_stacks_router() -> Router<DaemonState> {
+    Router::new()
+        .route(
+            ALMOST_FILLED_STACKS_PATH,
+            get(almost_filled_stacks_fraction_get),
+        )
+        .route(
+            &format!("{ALMOST_FILLED_STACKS_PATH}/:id"),
+            get(almost_filled_stacks_get),
+        )
+}
+
+/// Retrieves all stacks that are filled above a specified fraction threshold for all registered nodes.
+#[utoipa::path(
+    get,
+    path = "/{fraction}",
+    params(
+        ("fraction" = f64, Path, description = "The fraction threshold (0.0 to 100.0) to filter stacks")
+    ),
+    responses(
+        (status = 200, description = "Stack objects that are filled above the specified fraction threshold", body = Vec<Stack>),
```

Comment:
Can we use `OK` and `INTERNAL_SERVER_ERROR` and similar, instead of the numeric values?

Reply:
100% agreed

Reply:
100% agreed